workspaces/ui-v2/src/types/endpoints.ts

```diff
@@ -0,0 +1,17 @@
+export interface IPathParameter {
```

Comment:
I couldn't think of a better name for this file - but it's kinda any _domain_ type (specifically for the FE)